### PR TITLE
Preselect root images if present

### PR DESCRIPTION
**Pull request by Cryaaa**
Simple Fix I found for Issue #4. Furthermore a bug is fixed which was present when the function operating on the nodes of a workflow had multiple input images.

### CODE REVIEW

**Comment by haesleinhuepf** [comment]
## [Codecov](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/pull/5?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase) Report
Attention: Patch coverage is `0%` with `18 lines` in your changes missing coverage. Please review.
> Project coverage is 31.54%. Comparing base [(`9efe379`)](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/commit/9efe37919de0a3053529a00fbe60a5faa309cb85?dropdown=coverage&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase) to head [(`7859ff9`)](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/commit/7859ff9afc7c9b6f8d14ca17e455157366b17eb4?dropdown=coverage&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase).
> Report is 240 commits behind head on main.

| [Files with missing lines](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/pull/5?dropdown=coverage&src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase) | Patch % | Lines |
|---|---|---|
| [napari\_assistant/\_workflow\_io\_utility.py](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/pull/5?src=pr&el=tree&filepath=napari_assistant%2F_workflow_io_utility.py&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase#diff-bmFwYXJpX2Fzc2lzdGFudC9fd29ya2Zsb3dfaW9fdXRpbGl0eS5weQ==) | 0.00% | [18 Missing :warning: ](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/pull/5?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase) |

<details><summary>Additional details and impacted files</summary>


```diff
@@            Coverage Diff             @@
##             main       #5      +/-   ##
==========================================
- Coverage   32.25%   31.54%   -0.71%     
==========================================
  Files          12       12              
  Lines         803      821      +18     
==========================================
  Hits          259      259              
- Misses        544      562      +18     
```

</details>

[:umbrella: View full report in Codecov by Sentry](https://app.codecov.io/gh/haesleinhuepf/napari-assistant/pull/5?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase).   
:loudspeaker: Have feedback on the report? [Share it here](https://about.codecov.io/codecov-pr-comment-feedback/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Robert+Haase).

<details><summary> :rocket: New features to boost your workflow: </summary>

- :snowflake: [Test Analytics](https://docs.codecov.com/docs/test-analytics): Detect flaky tests, report on failures, and find test suite problems.
</details>

File: napari_assistant/_workflow_io_utility.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def initialise_root_functions(workflow, viewer):
     """
     # find all workflow steps with functions which have root images as an input
     root_functions = wf_steps_with_root_as_input(workflow)
+    layer_names = [str(lay) for lay in viewer.layers]
```
Comment (haesleinhuepf) [inline napari_assistant/_workflow_io_utility.py:23]: Hi Ryan @Cryaaa ,

just a quick question: After our discussion this week, do you want to change this to `layer.name` ?
```suggestion
    layer_names = [lay.name for lay in viewer.layers]
```

Comment (Cryaaa) [reply napari_assistant/_workflow_io_utility.py:23]: Ahhh yes I think I might have changed it in a different branch by accident! I’ve checked that it works already though so you can merge it with the change you have suggested!

**Review by haesleinhuepf — APPROVED** [review]
lgtm

**Pull request merged**